src/services/request.service.js

```diff
@@ -12,12 +12,13 @@ export default class RequestService {
    * @returns {Object} Saved trip request details
     */
   static handleTripRequest(tripRequestData) {
+    console.log(tripRequestData);
```

Comment:
Maybe you forgot to remove this `console`?

Reply:
Yes for sure, I had removed it